mne/viz/topo.py

```diff
@@ -694,7 +694,7 @@ def plot_topo_image_epochs(epochs, layout=None, sigma=0., vmin=None,
         Figure distributing one image per channel across sensor topography.
     """
     scalings = _handle_default('scalings', scalings)
-    data = epochs.get_data()
+    data = epochs.get_data().copy()
```

Comment:
Instead of copying here, it could be much more memory efficient to only copy when necessary. Is it possible to do the copy on an epoch-by-epoch basis before plotting?